x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group2/execution_status.ts

```diff
@@ -183,44 +183,6 @@ export default function executionStatusAlertTests({ getService }: FtrProviderCon
       await ensureAlertUpdatedAtHasNotChanged(alertId, alertUpdatedAt);
     });
 
-    it('should eventually have error reason "validate" when appropriate', async () => {
```

Comment:
I'm curious why we removed this test.  Also, shouldn't we have some action and alerting tests for the new bits?  I suspect they may be hard to build, perhaps impossible with only 1 Kibana?  In any case, feels like we should have _something_, and I think it's fine to add as a separate PR before FF.

Reply:
This was invalid for the previous version of this PR. 
Added back.